content/tools/api/AboutRestAPI.md

```diff
@@ -9,22 +9,29 @@ Contrast has a few API versions:
 * Application APIs (v2) 
 * HATEOS-compliant REST API (v3) **Recommended**
 
->**Note:** We haven't officially deprecated v1 or v2, but strongly recommend using v3.
+>**Note:** Contrast hasen't officially deprecated v1 or v2, but strongly recommend using v3.
 
 ## Overview
-The Contrast plugin allows you to collect nearly all the same information that is accessible through the Contrast UI. You can get vulnerability data, application, server and library information, download agents and much more.
 
-The API is available to both **SaaS** and **Enterprise On-Premises** users. The [endpoint for SaaS users](https://app.contrastsecurity.com/Contrast/api) is the same for everyone, while for On-Premise users it will depend on where the TeamServer application is installed on the client's network.
+The Contrast plugin allows you to collect nearly all the same information that is accessible through the Contrast interface - dowload agents, get vulnerability, application, server and library information, and much more. Contrast's goal is to make sure Contrast offers data to allow complete integration into your existing Continuous Integration (CI), SIEM software, or other business intelligence dashboards.
```

Comment:
@kaitlynmortimer typo: "download" - Also "Contrast's goal is to make sure Contrast..." sounds bad to me - can one of these switch? "Our goal is to make sure Contrast..." or "Contrast's goal is to make sure we..."